walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,135 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
+  /*
+   * Assigns attendees from request to local HashMap to search for them in constant time.
+   */
+  private HashSet<String> toSet(MeetingRequest request) {
```

Comment:
This specifically adds attendees to a set, while MeetingRequest has other data in it. Can you clarify this in the method name?

Reply:
Done!